tests/test_integration_upload.py

```diff
@@ -36,16 +52,17 @@ def test_upload_file():
     print("File upload successful, sialink: " + sialink2)
 
     headers = responses.calls[0].request.headers
-    assert headers["Content-Type"].startswith("multipart/form-data;")
+    assert headers["Content-Type"]
```

Comment:
q: Why did this change?

Reply:
requests doesn't do multipart streaming upload, so single file uploads are no longer multipart